plugins/algebra/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 #
 # Mozilla Public License Version 2.0
 
+# Temporary setting for the detray::scalar type, until it can be removed.
+set( DETRAY_CUSTOM_SCALARTYPE "double" CACHE STRING
```

Comment:
so `double` is default scalar type?

Reply:
That has been the status quo so far. :smile:

We should probably change it, but since that was the default all this time, I didn't want to change that in this PR. (The CI has a separate build setup for using `float`. That would also need to change if we wanted to change the default here.) And in any case, once this project is also updated not to rely on choosing the scalar type as a CMake configuration parameter, I expect that we'll be building all of the tests with `float` and `double` precision all the time anyway.